Contributors-Guide/Bug-Triage.md

```diff
@@ -75,6 +68,9 @@ In addition to manually checking Bugzilla for bugs to triage, it is also
 possible to receive emails when new
 bugs are filed or existing bugs get updated.
 
+Plus weekly basis(Monday) there will be mail about new untriaged bugs
```

Comment:
The `Plus` sounds out of place. This can be improved as:

`There will be a weekly (Monday) mail sent to gluster-devel list mailing list about new untriaged bugs`

Reply:
Each week, on the Monday, an email is sent to gluster-devel about new bugs which are yet to be triaged.